app/javascript/glitch/components/status/index.js

```diff
@@ -664,6 +664,21 @@ backgrounds for collapsed statuses are enabled.
       ) background = attachments.getIn([0, 'preview_url']);
     }
 
+/*
+
+Here we prepare extra data-* attributes for CSS selectors.
+Users can use those for theming, hiding avatars etc via UserStyle
+
+*/
+
+    let selectorAttribs = {
+      'data-status-by': '@' + status.getIn(['account', 'acct']),
+    };
+
+    if (prepend && account) {
+      selectorAttribs['data-'+prepend+'-by'] = '@' + account.get('acct');
```

Comment:
right, I'm doing some changes still anyway

I tried to have `{template-string: value}` and it cried wolf, so I tried with +, it was still bad, and then I left it like that xD